### PR TITLE
Fix mouse events

### LINKED ISSUE #97: [BUG] Windows jump when resizing
1. open an application
2. snap it to any predefined wintile area
3. try to resize from any corner/edge
4. watch the window jump oddly

Expected behavior is that height or width be maintained depending on if x or y is changing.

[Screencast from 2023-04-25 10-59-27.webm](https://user-images.githubusercontent.com/3096423/234227678-d791943d-3c83-42f4-8e3a-0fefb52e395b.webm)

### PR DESCRIPTION
**Pull request by rsxchin**
resolves #97

### CODE REVIEW

**Comment by rsxchin** [comment]
@phavekes, did you ever accept the maintainer position? If you did, would you look over my 2 PRs?

File: extension.js [hunk 1]
```diff
@@ -151,9 +151,8 @@ function moveApp(app, loc) {
 	app.wintile.row = loc.row;
 	app.wintile.width = loc.width;
 	app.wintile.height = loc.height;
-
 	let window = app.get_frame_rect()
-	_log("window.x: "+window.x+" window.y: "+window.y+" window.width: "+window.width+" window.height: "+window.height)
+	_log("moveApp) window.x: " + window.x + " window.y: " + window.y + " window.width: " + window.width + " window.height: " + window.height)
```
Comment (phavekes) [inline extension.js:155]: an ")" extra

**Comment by rsxchin** [comment]
That's merely a string ")" to conform with the other logs.

**Comment by phavekes** [comment]
When trying your modified extension.js, I get an error 
`expected expression, got ')'`
You are right that the log string can't be the issue, but I don't see where the error emits

**Comment by rsxchin** [comment]
I swapped this kne to draft and I'll track it down tomorrow. The other PR is more important. Let me know what you think

**Comment by Fmstrat** [comment]
@rsxchin As you are pretty active in PRs, would you like to be added as a maintainer officially? I have not been able to dedicate the appropriate amount of time here, so I'm sure @phavekes would appreciate the help. Thanks!

**Comment by rsxchin** [comment]
I'm closing this pr in favor of another smaller one

**Pull request closed without merging**